project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("org.scalajs.junit.JUnitFramework"),
+        testOptions += Tests.Argument(
+          new TestFramework("org.scalajs.junit.JUnitFramework"), "-v", "-a"),
```

Comment:
And most importantly, so that users who cross-compile their JUnit test need not juggle with `testOptions` for two different frameworks.